RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -2356,5 +2346,127 @@ private boolean isDomainIdInSAMList(String osAccountSID) {
+                    if (systemRoot != null && !systemRoot.isEmpty()) {
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_PATH, parentModuleName, systemRoot));
+                    }
+                    if (productId != null && !productId.isEmpty()) {
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_PRODUCT_ID, parentModuleName, productId));
+                    }
+                    if (regOwner != null && !regOwner.isEmpty()) {
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_OWNER, parentModuleName, regOwner));
+                    }
+                    if (regOrg != null && !regOrg.isEmpty()) {
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_ORGANIZATION, parentModuleName, regOrg));
+                    }
+                    
+                    postArtifact(createArtifactWithAttributes(BlackboardArtifact.Type.TSK_OS_INFO, context.getDataSource(), bbattributes));
+                }
+            } catch (TskCoreException ex) {
+                logger.log(Level.SEVERE, "Failed to create default OS_INFO artifact", ex); //NON-NLS
+            }
+        }
+
+        void setCompName(String compName) {
+            if(this.compName == null || this.compName.isEmpty()) {
+                this.compName = compName;
+            }
+        }
+
+        void setOsName(String progName) {
+            if(this.progName == null || this.progName.isEmpty()) {
+                this.progName = progName;
+            }
+        }
+
+        void setProcessorArchitecture(String processorArchitecture) {
+            if(this.processorArchitecture == null || this.processorArchitecture.isEmpty()) {
+                this.processorArchitecture = processorArchitecture;
+            }
+        }
+
+        void setTempDir(String tempDir) {
+            if(this.tempDir == null || this.tempDir.isEmpty()) {
+                this.tempDir = tempDir;
+            }
+        }
+
+        void setDomain(String domain) {
+            if(this.domain == null || this.domain.isEmpty()) {
+                this.domain = domain;
+            }
+        }
+
+        void setInstalltime(Long installtime) {
+            if(this.domain == null) {
+                this.installtime = installtime;
+            }
+        }
+
+        void setSystemRoot(String systemRoot) {
+            if(this.systemRoot == null || this.systemRoot.isEmpty()) {
+                this.systemRoot = systemRoot;
+            }
+        }
+
+        void setProductId(String productId) {
+            if(this.productId == null || this.productId.isEmpty()) {
+                this.productId = productId;
+            }
+        }
+
+        void setRegOwner(String regOwner) {
+            if(this.regOwner == null || this.regOwner.isEmpty()) {
+                this.regOwner = regOwner;
+            }
+        }
+
+        public void setRegOrg(String regOrg) {
```

Comment:
This could probably be package private.

Reply:
Is there an issue taking out the `public` modifier for `setRegOrg` or is that needed for something?